docs/features/README.md

```diff
@@ -1,11 +1,10 @@
 # Overview
 
-Besides Connaisseur's central functionality, several additional features are available:
+Besides Connaisseur's central functionality, several additional features are available, such as:
 
+- [Alerting](./alerting.md): *send alerts based on verification result*
 - [Detection Mode](./detection_mode.md): *warn but do not block invalid images*
 - [Namespaced Validation](./namespaced_validation.md): *restrict validation to dedicated namespaces*
-- [Alerting](./alerting.md): *send alerts based on verification result*
-- [Automatic Child Approval](automatic_child_approval.md): *configure approval of Kubernetes child resources*
```

Comment:
Why no more ACA? If due to experimental, wouldn't an experimental feature list be the better solution than to drop?

Reply:
Thought I switched from "this is the list" (where the list was incomplete) to "feature such as" (which is a selective list of interesting features. But needs adjustment either way. Central readme as well

Reply:
indeed I did add a "such as" above and thus it is merely a list of "some featured features"